transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/internal/ExternalPrivacyContext.java

```diff
@@ -0,0 +1,25 @@
+package com.google.android.datatransport.cct.internal;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class ExternalPrivacyContext {
+  @Nullable
+  public abstract ExternalPRequestContext getPrequest();
```

Comment:
This and the setter should probably both be `getPRequest` if we're sticking with that name.

Reply:
Discussed offline